tutorials/2d/custom_drawing_in_2d.rst

```diff
@@ -139,9 +139,74 @@ call ``queue_redraw()`` from the ``_process()`` callback, like this:
         }
     }
 
+Coordinates
+-----------
+
+The drawing API uses the CanvasItem's coordinate system, not necessarily pixel
+coordinates. Which means it uses the coordinate space created after applying
+the CanvasItem's transform. Additionally, you can use
+:ref:`draw_set_transform<class_CanvasItem_method_draw_set_transform>` or
+:ref:`draw_set_transform_matrix<class_CanvasItem_method_draw_set_transform_matrix>`
+to apply a custom transform.
```

Comment:
```suggestion
The drawing API uses the CanvasItem's coordinate system, not necessarily pixel
coordinates. Which means it uses the coordinate space created after applying
the CanvasItem's transform. Additionally, you can apply a custom transform on
top of it by using
:ref:`draw_set_transform<class_CanvasItem_method_draw_set_transform>` or
:ref:`draw_set_transform_matrix<class_CanvasItem_method_draw_set_transform_matrix>`.
```
(not sure if it should be formatted like I've suggested)

I'd reword it to make it more obvious that the additional custom transform is applied on top of the original transform (instead of e.g. replacing it).

cc @clayjohn to tell if it looks fine for you.

Reply:
Looks fine to me! I agree that it is more clear